.github/workflows/health-sdk.check.yml

```diff
@@ -36,43 +36,14 @@ jobs:
            cd HealthSDK/GiniHealthSDK
            swift package update
 
-    - name: Check package GiniHealthSDKPinning
-      run: |
-           cd HealthSDK/GiniHealthSDKPinning
-           swift package update
-
     - name: Build sdk targets
       run: |
         xcodebuild -workspace GiniMobile.xcworkspace -scheme "${{ matrix.target }}" -destination "${{ matrix.destination }}" CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO ONLY_ACTIVE_ARCH=NO
 
     - name: Run unit tests
       run: |
         xcodebuild clean test -workspace GiniMobile.xcworkspace -scheme "${{ matrix.target }}Tests" -destination "${{ matrix.destination }}" CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO ONLY_ACTIVE_ARCH=NO
-   
+
     - name: Build example app
       run: |
-        xcodebuild -project HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample.xcodeproj -scheme "GiniHealthSDKPinningExample" -destination "${{ matrix.destination }}" CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO ONLY_ACTIVE_ARCH=NO    
-
-    - name: Build pinning example app and run integration tests
```

Comment:
[Suggestion]: @antosj we need to run pinning-related tests just using the GiniHealthSDKExampleTests scheme